tests/unit/test_routes.py

```diff
@@ -371,6 +371,66 @@ def test_update_resource(module_client, module_db, fake_auth_from_oc):
     assert (response.status_code == 404)
 
 
+def test_search(module_client, module_db, fake_auth_from_oc):
```

Comment:
Unit tests should not test outside services. Let's mock the Algolia API calls like we do with the auth from oc here

Reply:
Oh, and I'm glad we discovered this. We're not mocking the indexing of creating and updating the resources in the other tests, are we? We need to mock algolia in any test that performs an algolia operation.